docs/usage-examples/updateOne.txt

```diff
@@ -0,0 +1,109 @@
+.. _laravel-update-one-usage:
+
+=================
+Update a Document
+=================
```

Comment:
I would add the `contents` for the TOC, and the tags while you're working on these pages.
```suggestion
=================

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: twocols

.. facet::
   :name: genre
   :values: reference
 
.. meta::
   :keywords: <whatever you think>
```